### PR TITLE
Support additional parameters for infinityModel()

### PR DESCRIPTION
**Pull request by bruce**
Additional options can be provided in the object passed to
`InfinityRoute`'s `infinityModel()` for use by `DS.Store.find()`.
The `perPage` and `startingPage` properties will still be passed as
`per_page` and `page` params, respectively.

Acceptance tests use a new `/category/:category` route in the
dummy application.

For #6.

### CODE REVIEW

**Comment by bruce** [comment]
Note: submitted this a bit prematurely; still adding support for `_loadingMore` use of `find()`. Working on this (and hopefully a good test).

**Comment by bruce** [comment]
This is now ready for review.

File: addon/mixins/route.js [hunk 1]
```diff
@@ -108,48 +121,53 @@ export default Ember.Mixin.create({
...
+        Ember.run.scheduleOnce('afterRender', _this, 'infinityModelLoaded', { totalPages: totalPages });
       }
-      return false;
+    }
+    return false;
+  },
+
+  actions: {
+    infinityLoad: function() {
+      this._infinityLoad();
```
Comment (hhff) [inline addon/mixins/route.js]: nice!

**Comment by hhff** [comment]
Really really nice!  thankyou @bruce !

If you can squash this into a single commit - and add something to README to reflect this change, I'll merge and release asap

**Comment by bruce** [comment]
Thanks! I'll get that in tonight/tomorrow morning.

**Comment by bruce** [comment]
@hhff Squashed and README content added. Thanks!

**Pull request merged**